src/style.css

```diff
@@ -1 +1,45 @@
 /* add styles here */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
Avoid using the `*` selector as it impacts performance. It's better to set styles only for elements that require them. You can find this in the task checklist under the CSS section.